netprofile_tickets/netprofile_tickets/__init__.py

```diff
@@ -299,6 +299,12 @@ def get_settings(self, vhost='MAIN', scope='global'):
                                         'updating them.'),
                             type='bool',
                             default=True),
+                    Setting('notify_on_assign',
+                            title=_('Notify on ticket assignment'),
+                            help_text=_('Notify on ticket assignment on '
```

Comment:
Better help text might be: "Send notification when a ticket has been assigned to me or my group."